src/hotspot/share/opto/macro.hpp

```diff
@@ -82,14 +82,7 @@ class PhaseMacroExpand : public Phase {
 
 private:
   // projections extracted from a call node
-  ProjNode *_fallthroughproj;
-  ProjNode *_fallthroughcatchproj;
-  ProjNode *_ioproj_fallthrough;
-  ProjNode *_ioproj_catchall;
-  ProjNode *_catchallcatchproj;
-  ProjNode *_memproj_fallthrough;
-  ProjNode *_memproj_catchall;
-  ProjNode *_resproj;
+  CallProjections _call_projects;
```

Comment:
Please, use either `_call_projs` or `_call_projections`.

Reply:
change to _callprojs because I found other places use callprojs.